packages/@webex/plugin-meetings/src/meeting/voicea-meeting.ts

```diff
@@ -118,3 +121,41 @@ export const processNewCaptions = ({
   }
   transcriptData.interimCaptions[transcriptId] = interimTranscriptionIds;
 };
+
+export const processHighlightCreated = ({
```

Comment:
We simply process the highlights voicea gives back and pack it into the meetings object as an array.